manifests/a/Azul/Zulu/10/JDK/10.3.5/Azul.Zulu.10.JDK.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.4.1 Dumplings Mod $debug=QUSU.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Azul.Zulu.10.JDK
+PackageVersion: 10.3.5

```

```suggestion
PackageVersion: "10.3"
```
Please update file path to match this change.